core/cap-TBD.md

```diff
@@ -0,0 +1,562 @@
+flexible in adapting to new application needs, and wish to be able to submit
+custom turing-complete code to run in the transaction-execution phase of the
+network.
+
+The Soroban project provides such an ability, through mechanisms specified in
+the various sub-CAPs linked from this CAP. Each sub-CAP specifies its
+relationship to the whole Soroban project and, where necessary, links to other
+sub-CAPs that it depends on or interacts with.
+
+### Goals Alignment
+
+This CAP is aligned with the following Stellar Network Goals:
+
+  - The Stellar Network should make it easy for developers of Stellar projects
+    to create highly usable products
+
+## Abstract
+
+Soroban adds a platform for execuiting smart contracts to the Stellar network.
+
+It is small, simple, efficient, and based on standardized technology, leveraging
+standard tools and techniques whenever possible, and focusing on providing a
+high-quality and low-effort developer experience for writing smart contracts.
+
+## Specification
+
+All specifications _besides_ the cumulative XDR diffs below are provided in
+the following sub-CAPs:
+
+  - [CAP-0046 - Smart Contract Runtime Environment](./cap-0046.md)
+    covers the code and data _environment_ that smart contracts run inside,
+    rather than their relationship to the rest of the network. This mostly
+    relates to the new XDR files below, rather than the diffs.
+  - [CAP-0047 - Smart Contract Lifecycle](./cap-0047.md) covers the mechanism
+    for creating smart contracts _and_ changes to the ledger to store
+    global configuration parameters. 
+  - CAPs 0048, 0049 and 0050 are abandoned explorations of interoperability
+    between smart contracts and existing assets on the Stellar network.
+  - [CAP-0051 - Smart Contract Host Functions](./cap-0051.md) covers the new
+    functions provided by the smart contract host to contracts. The semantics of
+    some of these are specified in other topic-specific CAPs, but the central
+    list of them will continue to live in CAP-0051.
+  - [CAP-0052 - Smart Contract Interactions (miminal)](./cap-0052.md) covers
+    the invocation interface for contracts _and_ sketches a standard token
+    contract interface and authentication pattern.
+  - [CAP-0053 - Smart Contract Data](./cap-0053.md) covers new ledger
+    entries to store contract data.
+  - [CAP-0054 - Smart Contract Standardized Asset](./cap-0054.md) covers the
+    built-in token contract, that can also "wrap" existing Stellar assets.
+  - [CAP-0055 - Fee Model in Smart Contracts](./cap-0055.md) covers changes
+    to the network's fee-charging system to account for smart contracts.
+  - [CAP-0056 - Smart Contract Logging](./cap-0056.md) covers changes to
+    the transaction result, metadata, and ledger close metadata XDR to
+    capture a new set of "events" emitted during smart contract execution.
+  
+### XDR changes
+
+There are three entirely new XDR files:
```

Comment:
Should we consider pasting the actual XDR contents of these files in this CAP, rather than putting them in the contents folder? 
I think there is value having them rendered in the actual doc since they are part of the XDR diff. Putting in a separate folder feels like they are auxiliary.

Reply:
I think they clutter the CAP rendering them inline. It's already a bit overwhelming to see all the diffs for files that were changed, and there's no diff-rendering advantage to doing them inline (every line starts with `+` and is colored green).